scripts/storage-initializer/README.md

```diff
@@ -0,0 +1,30 @@
+storage initializer download script
+
+### usage
+1. s3 with public access:
+```shell
+python3 download.py s3://models/classification/model.tar.gz /tmp/models/
+# we then download model.tar.gz and extract it into /tmp/models/
```

Comment:
how to download sample?

Reply:
Will I add docs about this with issue #54 fixed

Reply:
added